packages/block-editor/README.md

```diff
@@ -1032,6 +1032,10 @@ _Returns_
 
 A hook used to set the editor mode to zoomed out mode, invoking the hook sets the mode.
 
+_Parameters_
+
+-   _zoomOut_ `boolean`: If we should enter into zoomOut mode or not
+
```

Comment:
Previously the `useZoomOut` hook toggled between zoom in/out mode each time it ran. Passing a param of true/false allows us to intentionally choose if we want to use zoom out mode or not. 

This param wasn't necessary in the previous set-up because the pattern category panel would unmount (turning zoom out mode off) then re mount a new category (turning zoom out mode back on). There's [more context in this comment.](https://github.com/WordPress/gutenberg/pull/60257/files#r1553840557)